cf/flags/flags_test.go

```diff
@@ -73,7 +73,7 @@ var _ = Describe("Flags", func() {
 				Ω(fCtx.Bool("skip2")).To(Equal(true), "skip2 should be true")
 				Ω(fCtx.Bool("name")).To(Equal(false), "name should be false")
 				Ω(fCtx.String("name")).To(Equal("johndoe"), "name should be johndoe")
-				Expect(fCtx.Bool("non-exisit-flag")).To(Equal(false))
+				Expect(fCtx.Bool("non-exist-flag")).To(Equal(false))
```

Comment:
I'd normally want to change this to `nonexistent`, but I'm not sure if `non` is magical here.